### ISSUE TITLE
'Something went wrong' on identity server

### ISSUE THREAD

**Issue by DLawla** [XeroAPI/xero-oauth2-omniauth-strategy#5]
Hi guys,
Thanks for making a first pass at a strategy for connecting to Xero.
I was just having a play, and cloned the demo app and dropped in our secret/client ID (after configuring per this blog post: https://devblog.xero.com/xero-oauth-2-with-ruby-313a6ea37989), and hit this page:
![image](https://user-images.githubusercontent.com/12427222/71851231-f1a1d300-313a-11ea-9d0f-7127cd403487.png)
Notice that there is a 500 error at the bottom of the page.
As a sanity check, here is the redirect URI I entered (same as blog post):
![image](https://user-images.githubusercontent.com/12427222/71851415-683ed080-313b-11ea-9fd2-e02e85a27a6d.png)
As a sanity check, I ran this in a default Chrome browser and in incognito just in case and had the same outcome.
Before I did any deep dives, I just wanted to check in with you to see if there have been any updates not implemented into the gem, or any insights into why this could be happening.
Thanks again!

**Comment by DLawla** [comment]
Apologies, I have sorted this out. It turns out the redirect URI I entered when configuring the app should have been using `http`. A silly mistake. Might be helpful to update the blog post w/ a note on that.

Anywho, I would love to see a better error response beside the blanked 'something went wrong' as I have seen w/ other identity servers that can give you a hint about a bad/non-matching redirect uri.

Cheers guys
